server/src/main/java/org/opensearch/tasks/CancellableTask.java

```diff
@@ -50,6 +50,14 @@ public abstract class CancellableTask extends Task {
     private volatile String reason;
     private final AtomicBoolean cancelled = new AtomicBoolean(false);
     private final TimeValue cancelAfterTimeInterval;
+    /**
+     * The time this task was cancelled as a wall clock time since epoch ({@link System#currentTimeMillis()} style).
+     */
+    private long cancelledAt = -1;
+    /**
+     * The time this task was cancelled as a relative time ({@link System#nanoTime()} style).
+     */
+    private long cancelledAtNanos = -1;
```

Comment:
Is this used anywhere?

Reply:
This will be used by next set of changes [here](https://github.com/opensearch-project/OpenSearch/pull/7642) -  where we need to calculate elapsed time using this.